### PR TITLE
Removes the debilitate projectile list

### PR DESCRIPTION
**Pull request by Rohesie**
Old code, its logic being superseded.

This requires #3810 to compile properly.

### CODE REVIEW

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -1285,16 +1290,16 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
...
 	var/danger_message = "<span class='danger'>A glob of acid lands with a splat and explodes into noxious fumes!</span>"
 	armor_type = "bio"
 	accuracy_var_high = 10
 	max_range = 30
 
-/datum/ammo/xeno/boiler_gas/on_hit_mob(mob/M, obj/projectile/P)
-	drop_nade(get_turf(P), P.firer)
+/datum/ammo/xeno/boiler_gas/on_hit_mob(mob/living/victim, obj/projectile/proj)
+	drop_nade(get_turf(proj), proj.firer)
+	victim.Paralyze(4 SECONDS)
```
Comment (psykzz) [inline code/modules/projectiles/ammo_datums.dm:1302]: 4 seconds ? that feels a lot

Comment (Rohesie) [reply code/modules/projectiles/ammo_datums.dm]: Back to old behavior now.

Comment (jroinc) [reply code/modules/projectiles/ammo_datums.dm]: It's a direct hit, so...

**Comment by Rohesie** [comment]
Ready now.

**Pull request merged**